src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import { useCallback, memo } from 'react';
+import './Image.scss';
+
+// TODO Add loading on image onLoad
+
+export const ImageComponent = memo(({ src, size, onClick }) => {
+    const imageOnClick = useCallback(() => onClick(), [onClick]);
+
+    return (
+        <img
+            src={src}
+            className={`atom__image atom_image_${size}`}
```

Comment:
Still lack of testing, 
Missing underscores in class name (there is _ instead of __)

Reply:
done